src/gpuarray_blas_cuda_cublas.c

```diff
@@ -75,6 +75,7 @@ static const char *code_sgemvBH_T_a1_b1_small =                         \
   "  atomicAdd(&y[p][i*incy], yi);"                                     \
   "}\n";
 
+#if CUDA_VERSION < 8000
```

Comment:
if we compile with cuda8 on a computer with just a titan Black and a GT 520, we got those type of errors:

```
======================================================================
ERROR: pygpu.tests.test_blas.test_gemv((100, 128), 'float32', 'f', False, False, 1, True, False)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Tmp/lisa/os_v5/anaconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/u/bastienf/.local/lib64/python2.7/site-packages/pygpu-0.2.1-py2.7-linux-x86_64.egg/pygpu/tests/support.py", line 39, in f
    func(*args, **kwargs)
  File "/u/bastienf/.local/lib64/python2.7/site-packages/pygpu-0.2.1-py2.7-linux-x86_64.egg/pygpu/tests/test_blas.py", line 62, in gemv
    overwrite_y=overwrite)
  File "pygpu/blas.pyx", line 72, in pygpu.blas.gemv (pygpu/blas.c:2087)
  File "pygpu/blas.pyx", line 27, in pygpu.blas.pygpu_blas_rgemv (pygpu/blas.c:1395)
GpuArrayException: ('No such file or directory', 6)
```

If we compile on a computer with 1080 with cuda 7.5, I have those errors:


```
$ DEVICE=cuda1  make test
Running tests...
Test project /u/bastienf/repos/libgpuarray/Build
    Start 1: test_types
1/6 Test #1: test_types .......................   Passed    0.02 sec
    Start 2: test_util
2/6 Test #2: test_util ........................   Passed    0.03 sec
    Start 3: test_array
3/6 Test #3: test_array .......................***Failed    0.55 sec
    Start 4: test_elemwise
4/6 Test #4: test_elemwise ....................***Failed    5.24 sec
    Start 5: test_error
5/6 Test #5: test_error .......................   Passed    0.02 sec
    Start 6: test_buffer
6/6 Test #6: test_buffer ......................   Passed    2.68 sec

67% tests passed, 2 tests failed out of 6

Total Test time (real) =   8.70 sec

The following tests FAILED:
	  3 - test_array (Failed)
	  4 - test_elemwise (Failed)
Errors while running CTest
Makefile:117: recipe for target 'test' failed
make: *** [test] Error 8
```

On 1080 with cuda8, it work. on Titan Black with cuda 7.5, it work.